website/members/migrations/0051_membership_study_long_until.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from django.utils import timezone
+from django.db import migrations, models
+
+
+def change_until(apps, schema_editor):
+    Membership = apps.get_model("members", "Membership")
+    members = Membership.objects.filter(until=None and __.length == __.MEMBERSHIP_STUDY)
```

Comment:
That's not django filter syntax😂

Reply:
Memberships don't hav e a length field either, only type, so it should be something like

```
Membership.objects.filter(until=None, type=Membership.MEMBER).update(study_long=True, until=<the next 20xx-09-01 based on timezone.now() or just hardcode 2025>)
```